MineLeapMod/src/com/nautigsam/mineleapmod/helpers/McObfuscationHelper.java

```diff
@@ -66,6 +66,7 @@ public static int keyCode(KeyBinding key)
 		case 164:
 			return new String[] { fieldOrFunctionName, candidates[0] };
 		case 172:
+		case 180:

```

Comment:
It's an integer representing the actual version of Minecraft. We use it because the obfuscated methods' names change from one major version to another. We are in 1.8.0 now so the integer is 180.